README.md

```diff
@@ -60,18 +60,22 @@ To ensure a successful deployment, you will want to ensure that:
    to actually install RPM or DEB packages on the nodes. If you do not have this
    capability and you want to protect the nodes, you have to install using a
    privileged container.
-1. CrowdStrike's Helm Operator is a project, not a product, and released to the
-   community as a way to automate sensor deployment to kubernetes clusters. The
-   upstream repository for this project is
-   [https://github.com/CrowdStrike/falcon-helm](https://github.com/CrowdStrike/falcon-helm).
-1. The Helm Operator is IN DEVELOPMENT AND NOT PRODUCTION READY, so do not use
-   it! Use Helm Charts instead.
 
-# Installation
+# Quick Note Regarding Support
+🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 
+ 
+This Helm chart is IN DEVELOPMENT, AND NOT (YET) SUPPORTED!!
+
+🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 🔥 🚒 
+
+On 24-FEB-2021, CrowdStrike announced (via [press release](https://www.crowdstrike.com/press-releases/advanced-threat-protection-for-cloud-and-container-workloads/)) technology previews are available for deploying Falcon into Azure Kubernetes Service (AKS), Google Kubernetes Engine (GKE), Rancher, and Red Hat OpenShift Container Platform (OCP).
 
-## Using the Helm Operator
```

Comment:
Can you keep the helm operator section? This is still needed since you will be able to deploy either a helm chart or a helm operator from this repo.

Reply:
The rationale behind the drop was there is no operator code /today/.

Reply:
That's actually not accurate... the code exists, and if you know what you are doing, can run it.

Reply:
Why you trying to say I don't know what I'm doing.

![giphy (5)](https://user-images.githubusercontent.com/5713754/113731814-10d53a80-96c7-11eb-8e4a-4e903954f099.gif)

Reply:
Updated